dom/automation/loopdomfuzz.py

```diff
@@ -238,4 +238,11 @@ def afterColon(s):
 
 
 if __name__ == "__main__":
+    try:
+        from xvfbwrapper import Xvfb
```

Comment:
I don't think we always want to run with Xvfb when xvfbwrapper is installed. I have it installed globally, so I would need to edit the file to run without it. I'd suggest a pattern like this:

    try:
        from xvfbwrapper import Xvfb
        HAVE_XVFB = True
    except ImportError:
        HAVE_XVFB = False
    
    def start_runs():
        many_timed_runs(....)
    
    if HAVE_XVFB and int(os.getenv("USE_XVFB", 0)):
        with Xvfb():
            start_runs()
    else:
        start_runs()